kolibri/utils/version.py

```diff
@@ -114,6 +114,11 @@
 
 logger = logging.getLogger(__name__)
 
+ordered_versions = ('alpha', 'beta', 'rc', 'final')
```

Comment:
Maybe `ORDERED_VERSIONS` is better. The reason that `logger` is lowercase is that it behaves as a module or class instance.